hopp/simulation/technologies/resource/nsrdb_data.py

```diff
@@ -0,0 +1,226 @@
+from rex import NSRDBX
+from rex.sam_resource import SAMResource
+import numpy as np 
+from hopp.simulation.technologies.resource.resource import Resource
+from typing import Optional, Union
+from pathlib import Path
+import os
+from hopp.utilities.validators import range_val
+NSRDB_DEP = "/datasets/NSRDB/deprecated_v3/nsrdb_"
+
+# NOTE: Current version of PSM v3.2.2 which corresponds to /api/nsrdb/v2/solar/psm3-2-2-download 
+NSRDB_NEW = "/datasets/NSRDB/current/nsrdb_"
```

Comment:
I'd recommend converting these to `Path` objects (`Path("file_path")`) to make some later manipulations easier.